asv_bench/benchmarks/benchmarks.py

```diff
@@ -11,7 +11,7 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""General modin benchmarks"""
+"""General modin benchmarks."""
```

Comment:
```suggestion
"""General Modin benchmarks."""
```

https://github.com/modin-project/modin/issues/2934#issuecomment-815494134